### PR TITLE
Js intention7

### PR DESCRIPTION
**Pull request by jordy1611**
### What’s this PR do?
* Individual buttons now change color when clicked and other buttons return to default color, only one button changes color. Minutes and seconds input fields only take in positive integers and arguments.

### Where should the reviewer start?
* Examine all of main.js. For changes to CSS start at activity-buttons & time-input

### How should this be manually tested?
* Click the three activity buttons quickly in different combinations. Try entering any letters or symbols into the input fields and click start activity button.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  button.classList.add("btn-default");
+  button.classList.remove("exercise-selected");
+  image.src = "assets/exercise.svg";
+}
+
+var timeInput = document.querySelector(".time-input");
+timeInput.addEventListener("keydown", noLetters);
+function noLetters(event) {
+    if ([69, 187, 188, 189, 190].includes(event.keyCode)) {
+      event.preventDefault();
```
Comment (NakiNorton) [inline main.js:69]: I tested this function out and it's working perfectly. Nice work!

**Review by NakiNorton — APPROVED** [review]
It's great to see the activity buttons changing color when clicked. Great work utilizing the add and remove methods.

File: index.html [hunk 2]
```diff
@@ -15,18 +15,22 @@ <h1>INTENTION TIMER</h1>
...
+          <button class="btn-default" id="meditate-button"><img class="meditate-img" src="assets/meditate.svg" alt="Image of lotus flower"><br/>
           Meditate</button>
-        <button class="exercise-button activity-buttons"><img src="assets/exercise.svg" alt="Image of shoes"><br/>
+          <button class="btn-default" id="exercise-button"><img class="exercise-img" src="assets/exercise.svg" alt="Image of shoes"><br/>
           Exercise</button>
+        </div>
         <label class="user-goal-prompt">What would you like to accomplish during this time?</label>
         <input class="description-input"/>
         <label class="minutes-prompt">Minutes</label>
         <label class="seconds-prompt">Seconds</label>
```
Comment (Abdeboskey) [inline index.html:29]: This is great, the added div around input fields and activity buttons is a nice touch, especially when it comes to formatting but using it for event targeting is clever too!

**Review by Abdeboskey — APPROVED** [review]
This is awesome progress, good to start to see some JS in this project, and I really like the way you are using helper functions to DRY things up. Also, the refactoring of the css looks really good, and helps things make more sense.

**Pull request merged**